frontend/zc_messaging/src/pages/message-board/index.jsx

```diff
@@ -365,14 +375,14 @@ const MessagingBoard = () => {
           <MessageWrapper>
             <MessageRoomViewHeader name={`#${roomName}`} />
             <MessageBoard
-              isLoadingMessages={isLoadingRoomMessages}
-              messages={roomChats || []}
+              isLoadingMessages={isFetching}
```

Comment:
I think adding both is fetching and is loading would be more beneficial. Because `isLoadingRoomMessages` only occurs when the component is mounted for the first time. Then the `isFetching`happens for all subsequent fetches that happen during the session when one is active on the application when its listening to any changes that occur when the data is updated

Reply:
The isLoadingRoomMessages trigger on the first load of the components. it doesn't trigger on channel change or on subsequent changes which makes it look as though it is not working

Reply:
Okay. I just re-read the docs and realised they updated the distinction between the two. Which is fine.
I guess isFetching can work when it's alone in this scenario

Reply:
OKay. Thanks